### PR TITLE
[rai] Added fromat name for edit_year

### PR DESCRIPTION
**Pull request by AlexZigante**
Added this format names for automatic library management purposes.
edit_year:  is the first original release year of movie or series.
info usable for "*Arr"s and other similar collection managers for imports

## Please follow the guide below

- You will be asked some questions, please read them **carefully** and answer honestly
- Put an `x` into all the boxes [ ] relevant to your *pull request* (like that [x])
- Use *Preview* tab to see how your *pull request* will actually look like

---

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check one of the following options:
- [ ] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [x] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Bug fix
- [ ] Improvement
- [ ] New extractor
- [x] New feature

---

### Description of your *pull request* and other information

Added this format name for automatic library management purposes.
edit_year:  is the first original release year of movie or series.
info usable for "*Arr"s and other similar collection managers for automatic imports

### CODE REVIEW

**Comment by AlexZigante** [comment]
hmm locally did work but i guess needs some fixes.. Anyone keen to help solve?

**Comment by Ashish0804** [comment]
@AlexZigante the issue is with code formatting. Run `flake8` on your file and fix the errors.

File: yt_dlp/extractor/rai.py [hunk 1]
```diff
@@ -339,6 +342,7 @@ def _real_extract(self, url):
             'episode': media.get('episode_title'),
             'episode_number': int_or_none(media.get('episode')),
             'subtitles': subtitles,
+            'edit_year': original_year,
```
Comment (Ashish0804) [inline yt_dlp/extractor/rai.py:345]: this is not a valid field name 
see common.py for valid fields

Comment (pukkandan) [reply yt_dlp/extractor/rai.py:345]: modified_date may be suitable here

File: yt_dlp/extractor/rai.py [hunk 2]
```diff
@@ -322,6 +322,9 @@ def _real_extract(self, url):
 
         alt_title = join_nonempty(media.get('subtitle'), media.get('toptitle'), delim=' - ')
 
+        edit_year = media.get('track_info').get('edit_year')
```
Comment (pukkandan) [inline yt_dlp/extractor/rai.py:325]: use traverse_obj/try_get

**Review by pukkandan — CHANGES_REQUESTED** [review]

**Pull request closed without merging**